CalorieTracker/CalorieTracker/Controllers/CalorieController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CalorieController.swift
+//  CalorieTracker
+//
+//  Created by Sean Acres on 8/23/19.
+//  Copyright © 2019 Sean Acres. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+class CalorieController {
+    func createCaloriesEntry(amount: String) {
+        guard let amountDouble = Double(amount) else { return }
+        let _ = CaloriesEntry(amount: amountDouble)
+        
+        saveToPersistentStore()
+    }
+    
+    func saveToPersistentStore() {
+        let moc = CoreDataStack.shared.mainContext
+        
```

Comment:
I think it's best to call `performAndWait` here because you really want this code to execute synchronously.